assets/sass/_functions.scss

```diff
@@ -2,6 +2,23 @@
 // Functions
 // ========================================================================= //
 
+// =========================================================================
+// Functions: Map deep get
+// ========================================================================= //
+
+/// Map deep get
```

Comment:
[Ummmmmm....](https://github.com/biglotteryfund/blf-alpha/blob/master/assets/sass/_functions.scss#L66)

Reply:
?

Reply:
Click the link! That function already exists in this file.

Reply:
lol